firefox_ui_tests/remote/security/test_green_larry.py

```diff
@@ -0,0 +1,107 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestGreenLarry(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://ssl-ev.mozqa.com/'
+
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        try:
+            self.windows.close_all([self.browser])
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_green_larry(self):
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.url)
+
+        # Get the information from the certificate
+        cert = self.browser.tabbar.tabs[0].certificate
+        address = self.security.get_address_from_certificate(cert)
+
+        # Check the identity popup label displays
+        org_label = self.identity_popup.organization_label
+        country_label = self.identity_popup.country_label
+        self.assertEqual(org_label.get_attribute('value'), cert['organization'])
+        self.assertEqual(country_label.get_attribute('value'), '(' + address['country'] + ')')
+
+        # Check the favicon
+        favicon = self.browser.navbar.locationbar.favicon
+        self.wait_for_condition(lambda _: favicon.get_attribute('hidden') == 'false')
+
+        # Check the identity popup box
+        identity_box = self.identity_popup.box
+        self.assertEqual(identity_box.get_attribute('className'), 'verifiedIdentity')
+
+        identity_box.click()
+
+        # Check the idenity popup doorhanger
+        doorhanger = self.identity_popup.popup
+        self.assertEqual(doorhanger.get_attribute('className'), 'verifiedIdentity')
+
+        # TODO: uncomment code when bug 1134872 lands
+        # Check that the lock icon is visible
+        # lock_icon = self.identity_popup.encryption_icon
+        # css_info_lock_icon = lock_icon.value_of_css_property('list-style-image')
+        # self.assertNotEqual(css_info_lock_icon, 'none')
+
+        # Check the host name
+        # Bug 443116 - Larry strips the 'www.' from the host name using the eTLDservice.
+        # This is expedted behavior.
+        host = self.identity_popup.host
+        common_name = self.marionette.execute_script("""
+          return Services.eTLD.getBaseDomainFromHost(arguments[0]);
+        """, script_args=[cert['commonName']])
```

Comment:
Please add a method to the Security class, which returns the domain. Keep in mind that [it is recommended](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Reference/Interface/nsIEffectiveTLDService#getBaseDomainFromHost%28%29) to use `getBaseDomain()`.